From 62fe9f9a3d21891e87883af2e0e3815155926b20 Mon Sep 17 00:00:00 2001
From: Akshay Hegde <code@akshay-hegde.com>
Date: Wed, 6 Jun 2018 21:11:29 -0700
Subject: [PATCH] Fix build issues on macOS 10.14

This fixes a build issue on macOS 10.14 (Mojave) where clang complained
that the header files `<config.h>` and `<pinentry-curses.h>` could not
be found.
---
 Source/AppDelegate.m                              | 2 +-
 Source/main.m                                     | 2 +-
 Source/pinentry-0.9.4/assuan/assuan-buffer.c      | 2 +-
 Source/pinentry-0.9.4/assuan/assuan-handler.c     | 2 +-
 Source/pinentry-0.9.4/assuan/assuan-listen.c      | 2 +-
 Source/pinentry-0.9.4/assuan/assuan-pipe-server.c | 2 +-
 Source/pinentry-0.9.4/assuan/assuan-util.c        | 2 +-
 Source/pinentry-0.9.4/pinentry/argparse.c         | 2 +-
 Source/pinentry-0.9.4/pinentry/password-cache.c   | 2 +-
 Source/pinentry-0.9.4/pinentry/pinentry-curses.c  | 2 +-
 Source/pinentry-0.9.4/pinentry/pinentry.c         | 2 +-
 Source/pinentry-0.9.4/secmem/secmem.c             | 2 +-
 Source/pinentry-0.9.4/secmem/util.c               | 2 +-
 13 files changed, 13 insertions(+), 13 deletions(-)

diff --git a/Source/AppDelegate.m b/Source/AppDelegate.m
index 1e44dc7..1bbc2cc 100644
--- a/Source/AppDelegate.m
+++ b/Source/AppDelegate.m
@@ -5,7 +5,7 @@
 #import "KeychainSupport.h"
 #import "pinentry.h"
 #ifdef FALLBACK_CURSES
-#include <pinentry-curses.h>
+#include "pinentry-curses.h"
 #endif
 
 
diff --git a/Source/main.m b/Source/main.m
index 292e752..3de01ab 100755
--- a/Source/main.m
+++ b/Source/main.m
@@ -47,7 +47,7 @@ BOOL isBundleValidSigned(NSBundle *bundle) {
 
 
 #ifdef FALLBACK_CURSES
-#import <pinentry-curses.h>
+#import "pinentry-curses.h"
 
 /* On Mac, the DISPLAY environment variable, which is passed from
  a session to gpg2 to gpg-agent to pinentry and which is used
diff --git a/Source/pinentry-0.9.4/assuan/assuan-buffer.c b/Source/pinentry-0.9.4/assuan/assuan-buffer.c
index 363d6cf..02c93c3 100644
--- a/Source/pinentry-0.9.4/assuan/assuan-buffer.c
+++ b/Source/pinentry-0.9.4/assuan/assuan-buffer.c
@@ -18,7 +18,7 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA
  */
 
-#include <config.h>
+#include "config.h"
 #include <stdlib.h>
 #include <string.h>
 #include <stdio.h>
diff --git a/Source/pinentry-0.9.4/assuan/assuan-handler.c b/Source/pinentry-0.9.4/assuan/assuan-handler.c
index aadad21..2b62cc9 100644
--- a/Source/pinentry-0.9.4/assuan/assuan-handler.c
+++ b/Source/pinentry-0.9.4/assuan/assuan-handler.c
@@ -18,7 +18,7 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA
  */
 
-#include <config.h>
+#include "config.h"
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
diff --git a/Source/pinentry-0.9.4/assuan/assuan-listen.c b/Source/pinentry-0.9.4/assuan/assuan-listen.c
index db63ad2..fdd6807 100644
--- a/Source/pinentry-0.9.4/assuan/assuan-listen.c
+++ b/Source/pinentry-0.9.4/assuan/assuan-listen.c
@@ -18,7 +18,7 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA
  */
 
-#include <config.h>
+#include "config.h"
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
diff --git a/Source/pinentry-0.9.4/assuan/assuan-pipe-server.c b/Source/pinentry-0.9.4/assuan/assuan-pipe-server.c
index c0d464f..16dd134 100644
--- a/Source/pinentry-0.9.4/assuan/assuan-pipe-server.c
+++ b/Source/pinentry-0.9.4/assuan/assuan-pipe-server.c
@@ -18,7 +18,7 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA
  */
 
-#include <config.h>
+#include "config.h"
 #include <stdlib.h>
 #include <stdio.h>
 
diff --git a/Source/pinentry-0.9.4/assuan/assuan-util.c b/Source/pinentry-0.9.4/assuan/assuan-util.c
index c2c899f..db0c8d6 100644
--- a/Source/pinentry-0.9.4/assuan/assuan-util.c
+++ b/Source/pinentry-0.9.4/assuan/assuan-util.c
@@ -18,7 +18,7 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA
  */
 
-#include <config.h>
+#include "config.h"
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
diff --git a/Source/pinentry-0.9.4/pinentry/argparse.c b/Source/pinentry-0.9.4/pinentry/argparse.c
index e31b67e..806a446 100644
--- a/Source/pinentry-0.9.4/pinentry/argparse.c
+++ b/Source/pinentry-0.9.4/pinentry/argparse.c
@@ -35,7 +35,7 @@
  */
 
 #ifdef HAVE_CONFIG_H
-#include <config.h>
+#include "config.h"
 #endif
 
 #include <stdio.h>
diff --git a/Source/pinentry-0.9.4/pinentry/password-cache.c b/Source/pinentry-0.9.4/pinentry/password-cache.c
index 70b33f4..0a0146e 100644
--- a/Source/pinentry-0.9.4/pinentry/password-cache.c
+++ b/Source/pinentry-0.9.4/pinentry/password-cache.c
@@ -18,7 +18,7 @@
  */
 
 #ifdef HAVE_CONFIG_H
-# include <config.h>
+# include "config.h"
 #endif
 
 #include <stdlib.h>
diff --git a/Source/pinentry-0.9.4/pinentry/pinentry-curses.c b/Source/pinentry-0.9.4/pinentry/pinentry-curses.c
index d25ab2e..fc27d81 100644
--- a/Source/pinentry-0.9.4/pinentry/pinentry-curses.c
+++ b/Source/pinentry-0.9.4/pinentry/pinentry-curses.c
@@ -19,7 +19,7 @@
    02111-1307, USA  */
 
 #ifdef HAVE_CONFIG_H
-#include <config.h>
+#include "config.h"
 #endif
 #include <assert.h>
 #ifdef HAVE_NCURSESW
diff --git a/Source/pinentry-0.9.4/pinentry/pinentry.c b/Source/pinentry-0.9.4/pinentry/pinentry.c
index 9519848..9928972 100644
--- a/Source/pinentry-0.9.4/pinentry/pinentry.c
+++ b/Source/pinentry-0.9.4/pinentry/pinentry.c
@@ -18,7 +18,7 @@
  */
 
 #ifdef HAVE_CONFIG_H
-#include <config.h>
+#include "config.h"
 #endif
 
 #ifndef HAVE_W32CE_SYSTEM
diff --git a/Source/pinentry-0.9.4/secmem/secmem.c b/Source/pinentry-0.9.4/secmem/secmem.c
index 7a3bd4b..bfab36b 100644
--- a/Source/pinentry-0.9.4/secmem/secmem.c
+++ b/Source/pinentry-0.9.4/secmem/secmem.c
@@ -19,7 +19,7 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA
  */
 
-#include <config.h>
+#include "config.h"
 #include <stdio.h>
 #include <stdlib.h>
 #ifndef HAVE_W32CE_SYSTEM
diff --git a/Source/pinentry-0.9.4/secmem/util.c b/Source/pinentry-0.9.4/secmem/util.c
index a47164a..2520796 100644
--- a/Source/pinentry-0.9.4/secmem/util.c
+++ b/Source/pinentry-0.9.4/secmem/util.c
@@ -17,7 +17,7 @@
  */
 
 #ifdef HAVE_CONFIG_H
-#include <config.h>
+#include "config.h"
 #endif
 
 #define _GNU_SOURCE 1
